mkdocs/docs/HPC/python.md

```diff
@@ -0,0 +1,164 @@
+# Python 
+
+This page provides information on how to use Python on the HPC systems.
+It shows how to load Python modules and Python packages, 
+and how to run Python scripts interactively and using job scripts.
+
+## Python modules
+
+The HPC systems provide a number of Python modules that you can load into your environment.
+To see a list of available Python modules, you can run:
+
+```bash
+$ module avail Python/
+```
+
+For example, to load Python 3.11, you can run:
+
+```bash
+$ module load Python/3.11.3-GCCcore-12.3.0
```

Comment:
I wouldn't recommend loading this directly, since this is deliberately kept minimal w.r.t. including Python packages (otherwise maintaining these modules is a small nightmare), I would recommend loading `Python-bundle-PyPI` instead, except for older Python versions (< Python 3.11, or modules installed with a toolchain older than `GCC*/12.3.0` or `*/2023a`). With the switch to RHEL9, we'll only have Python 3.11 and newer available via modules, so little room for mistakes when recommending `Python-bundle-PyPI`

Reply:
fix in [recommend using python bundles over standalone python modules](https://github.com/hpcugent/vsc_user_docs/pull/689/commits/9309c873d4507ddfa238015ba791c1151be817d7)